components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/src/main/java/org/wso2/carbon/identity/oauth2/grant/organizationswitch/OrganizationSwitchGrant.java

```diff
@@ -136,6 +147,16 @@ public boolean validateGrant(OAuthTokenReqMessageContext tokReqMsgCtx) throws Id
         return true;
     }
 
+    @Override
+    public OAuth2AccessTokenRespDTO issue(OAuthTokenReqMessageContext tokReqMsgCtx) throws IdentityOAuth2Exception {
+
+        String tokenBindingRef = "os_" + tokReqMsgCtx.getProperty("tokenBindingReference");
```

Comment:
Please add a comment by explaining the need of adding the suffix

Reply:
Fixed with [30583f1](https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/12/commits/30583f1da23c808e776138e060865e40b93b7b55)